content/en/agent/kubernetes/log.md

```diff
@@ -140,6 +140,8 @@ Each tab in sections below shows a different way to apply integration templates
 
 ### Configuration
 
+**Note:** As a best practice, Datadog recommends using unified service tagging when assigning tags. Unified service tagging ties Datadog telemetry, including logs, together through the use of three standard tags: `env`, `service`, and `version`. To learn how to configure your environment with unified tagging, refer to the dedicated [unified service tagging][4] documentation.
```

Comment:
I have mixed feelings about having such a section here.

Indeed `env` should be global at the agent level and we should not recommend to use container labels or annotations.
Also, starting this section with this big block insist on this and totally disregard the setting of the `source` value which is key for log integrations.

Which is why I would actually put a note at the end of the setup explaining that they can unify service between Logs and traces by following the link.

WDYT?